### DISCUSSION TITLE
Postgres default metric column

### DISCUSSION THREAD

**Discussion by inad9300** [grafana/grafana#54684] [Data Sources]
Similar to how columns named "time" are preselected as time columns, I would like to propose that columns named "metric" be preselected as metric columns.

![image](https://user-images.githubusercontent.com/3440331/188326085-40b1aac5-77c4-4168-a978-ec8083e02021.png)

**Comment by inad9300** [comment]
Is there any interest in this? :cry: It should be a rather simple change to make, yielding a nice quality of life improvement.

**Comment by gabor** [comment]
hi @inad9300 , thank you for the idea. unfortunately, the query-editor has changed a lot since then, and we are moving in a different direction,  to a simpler query-editor, which does not auto-select fields.
